frontend/src/metabase/components/EntityMenu.jsx

```diff
@@ -49,7 +49,11 @@ class EntityMenu extends Component {
     } = this.props;
     const { open, menuItemContent } = this.state;
     return (
-      <div className={cx("relative", className)}>
+      <div
+        className={cx("relative", className, {
```

Comment:
Wdyt of converting this into a styled component?

Then, `relative` could be set in the styled component declaration.

Also, we could pass `open` as a prop to the styled component, styled it there, and not use class-based css for that.

Reply:
done